lib/dep_graph.js

```diff
@@ -38,33 +38,33 @@ function createDFS(edges, leavesOnly, result) {
  */
 var DepGraph = exports.DepGraph = function DepGraph() {
   this.nodes = {};
-  this.outgoingEdges = {}; // Node name -> [Dependency Node name]
-  this.incomingEdges = {}; // Node name -> [Dependant Node name]
+  this.outgoingEdges = {}; // Node -> [Dependency Node]
+  this.incomingEdges = {}; // Node -> [Dependant Node]
 };
 DepGraph.prototype = {
```

Comment:
The rest of these changes don't really make sense

E.g. `If a node with the specified node already exists`

This function is adding a node with the specified name.